projects/media-viewer/src/lib/viewers/convertible-content-viewer/convertible-content-viewer.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output, ViewEncapsulation } from '@angular/core';
+import { ResponseType, ViewerException } from '../viewer-exception.model';
+import { select, Store } from '@ngrx/store';
+import * as fromStore from '../../store/reducers/document.reducer';
+import * as fromDocumentActions from '../../store/actions/document.action';
+import { Subscription } from 'rxjs';
+import * as fromSelectors from '../../store/selectors/document.selectors';
+import { AnnotationSet } from '../../annotations/annotation-set/annotation-set.model';
+import { filter } from 'rxjs/operators';
+
+@Component({
+  selector: 'mv-conversion-viewer',
+  templateUrl: './convertible-content-viewer.component.html',
+  encapsulation: ViewEncapsulation.None
+})
+export class ConvertibleContentViewerComponent implements OnInit, OnDestroy {
+
+  @Input() originalUrl;
+  @Input() downloadFileName: string;
+
+  @Input() height = 'calc(100vh - 32px)';
```

Comment:
this is already defaulted in media-viewer, so there's no need to default it here

Reply:
amended